src/main/java/org/jabref/gui/documentviewer/DocumentViewerControl.java

```diff
@@ -91,7 +102,8 @@ private void updateCurrentPage(ObservableList<DocumentViewerPage> visiblePages)
                 // Successful hit
                 inMiddleOfViewport = Optional.of(hit.getCell());
             }
-        } catch (NoSuchElementException exception) {
+        } catch (
```

Comment:
Currently IntelliJ has Problems with the catch blocks. It always creates a new line here on auto formatting, we are investigating. Please remove the newline manually.